README.md

```diff
@@ -10,37 +10,159 @@ This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-mac
 
 This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
 
+**It is compatible with all JetBrains products except Android Studio.**
+
 ![](docs/demo-workflow.gif)
 
 
-## Logging
+## How it works
 
-SLF4J logging in this plugin has the following categories:
+Git Machete IntelliJ Plugin is a port of a great console tool - [git-machete](https://github.com/VirtusLab/git-machete#git-machete) - into an IntelliJ plugin.
```

Comment:
Ok, similar comments as in https://github.com/mkondratek/git-machete-blog-post/pull/1/files#r463683311 (prompted by @pdolega's feedback - we need to give an average non-advanced git user a clear view as to what kind of benefit they will have from modifying their git-related habits)